pom.xml

```diff
@@ -24,7 +24,7 @@
 
     <groupId>pl.plajer</groupId>
     <artifactId>villagedefense</artifactId>
-    <version>4.0.2</version>
+    <version>4.0.1</version>
```

Comment:
Why did you downgrade version?

Reply:
I didn't touch it, I just went to the development branch.